src/prepareNodeConfig.ts

```diff
@@ -138,6 +142,12 @@ export function prepareNodeConfig({
     },
   };
 
+  if (parentChainBeaconRpcUrl) {
```

Comment:
maybe we should add runtime validation that this must be provided if the parent chain is an L1, not just in the example

Reply:
Added here: https://github.com/OffchainLabs/arbitrum-orbit-sdk/pull/76/commits/f6e550ee64b58b7d8cda537e25f1a5d76eea956d